### PR TITLE
Added a method to get errors for a document

### PR DESCRIPTION
**Pull request by duckduckgrayduck**
Added a straightforward method to get errors, added it to the tests, and bumped to 3.4.1

### CODE REVIEW

File: docs/documents.rst [hunk 1]
```diff
@@ -224,6 +224,14 @@ Document
       A boolean indicating whether or not you have the ability to save this
       document.
 
+   .. attribute:: errors
```
Comment (mitchelljkotler) [inline docs/documents.rst]: This should be `get_errors`

File: setup.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='python-documentcloud',
-    version='3.4.0',
+    version='3.4.1',
```
Comment (mitchelljkotler) [inline setup.py]: should be 3.5.0 - I don't think I have been super consistent with this, but the first number is for backward incompatible changes, second number is for backward compatible new features, and last number is for backward compatible bug fixes.

**Review by mitchelljkotler — CHANGES_REQUESTED** [review]

File: docs/documents.rst (hunk 1 above)
Comment (duckduckgrayduck) [reply docs/documents.rst]: so get_errors itself returns <bound method Document.get_errors of <Document: 23932186 - 7.13.23_01002>>
which means it is just printing the method. errors can be used, or get_errors() can be used, which I included in the most recent push. Let me know what you think

Comment (mitchelljkotler) [reply docs/documents.rst]: Yes, `get_errors` is a method and needs to be called.  I didn't see `errors` defined anywhere, but now remember I have it set up to automatically call `get_errors` if you just do `errors`.  That is what I get for being clever.  Documenting either one seems fine.

Comment (duckduckgrayduck) [reply docs/documents.rst]: Sounds good, I documented get_errors() in the most recent push

**Pull request merged**